modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp

```diff
@@ -57,7 +55,7 @@ kbSelectParentDisplayContext.populatePortletBreadcrumbEntries(currentURLObj);
 						).put(
 							"resourcePrimKey", kbSelectParentDisplayContext.getParentResourcePrimKey()
 						).put(
-							"title", parentTitle
+							"title", kbSelectParentDisplayContext.getContentSelectionTitle()
```

Comment:
Can you rename `getContentSelectionTitle` to `getSelectedEntryTitle`? Also, rename any variables in the display context to match this.

Sorry I didn't catch this before.

Reply:
I edited it! 
Thank you